plugin/src/test/java/io/jenkins/plugins/casc/SchemaGenerationTest.java

```diff
@@ -20,47 +27,45 @@
     @Rule
     public JenkinsConfiguredWithCodeRule j = new JenkinsConfiguredWithCodeRule();
 
-    @Test
-    public void schemaShouldSucceed() throws IOException {
-
-        /**
-         *Validate the schema against a validator
-         * or against the already defined schema.
-         */
-        String s = generateSchema();
-        System.out.println(s);
-        String fileName = "Schema";
-        BufferedWriter writer = new BufferedWriter(new FileWriter(fileName));
-        writer.write(s);
-        writer.close();
-    }
-
     @Test
     public void downloadOldSchema() throws Exception {
 
-        String fileName = "OldSchema";
         JenkinsRule.WebClient client = j.createWebClient();
         WebRequest request = new WebRequest(client.createCrumbedUrl("configuration-as-code/schema"), POST);
         WebResponse response = client.loadWebResponse(request);
-        BufferedWriter writer = new BufferedWriter(new FileWriter(fileName));
-        writer.write(response.getContentAsString()
-        );
-        writer.close();
+        System.out.println(response);
     }
 
 
     @Test
-    public void validateSchema() {
-
+    public void validateSchema() throws IOException {
+        String schema = generateSchema();
+        JSONObject jsonSchema = new JSONObject(
```

Comment:
@casz This method throws an error that the JSON Schema has duplicate keys, as you suggested earlier, we might have to get rid of the duplicates via creating json objects and then 
`@Override
    public JSONObject putOnce(String key, Object value) throws JSONException {`
override the putOnce method to get rid of duplicates.
Thougts on this approach.

Reply:
I think it's better if `generateSchema` returns a JSONObject and inside generate schema you will build up the schema using JSON methods instead of writing the JSON by hand.

Reply:
Perfect @casz :) Thanks.